### PR TITLE
Add browser test for translated programs

### PR DESCRIPTION
**Pull request by carolinedanzi**
### Description
1. Adds `admin_translations.ts` with helper methods for translating programs (and later questions)
2. Adds `admin_program-translation.test.ts` with two tests - one for translating a program's name and description, and one for viewing a toast message when a program is not fully translated

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
#862

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java [hunk 1]
```diff
@@ -150,6 +150,7 @@ private ContainerTag programCard(
             .withText(applyText)
             .withId(baseId + "-apply")
             .withClasses(
+                "apply-button",
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java:153]: Nit (non-blocking): You could define this constant in `view/style/ReferenceClasses.java`.

File: browser-test/src/support/admin_translations.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export class AdminTranslations {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async selectLanguage(language: string) {
+    await this.page.click(`.language-link:has-text("${language}")`);
```
Comment (hmhsu) [inline browser-test/src/support/admin_translations.ts:11]: Suggestion (blocking): please prefer `:text("search-text")` over `:has-text("search-text")` as the latter returns any parents that contain an element that has the matching text, while the former returns the immediate element that contains the matching text.

**Review by hmhsu — APPROVED** [review]
Thanks for adding the tests.

File: universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java (hunk 1 above)
Comment (carolinedanzi) [reply universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java:153]: Great suggestion! Updated

File: browser-test/src/support/admin_translations.ts (hunk 2 above)
Comment (carolinedanzi) [reply browser-test/src/support/admin_translations.ts:11]: Ah that's good to know, thank you!

**Comment by carolinedanzi** [comment]
> Thanks for adding the tests.

Thank you for the review!

File: browser-test/src/support/admin_translations.ts (hunk 2 above)
Comment (carolinedanzi) [reply browser-test/src/support/admin_translations.ts]: Done!

**Pull request merged**